src/controllers/dockerflow.js

```diff
@@ -2,10 +2,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+ import fs from 'fs'
+ import path from 'path'
+
+ import AppConstants from '../appConstants.js'
+ import { version, homepage } from '../../package.json'
+
+ const versionJsonPath = path.join(__dirname, 'version.json')
+ 
+ // If the version.json file already exists (e.g., created by circle + docker),
+ // don't need to generate it
+ if (!fs.existsSync(versionJsonPath)) { 
+   const versionJson = {
+     source: homepage,
+     version
```

Comment:
I imagine this could be useful, what do you think?

```suggestion
     version,
     NODE_ENV: process.env.NODE_ENV,
```

Reply:
Oh actually, I also realised that `version` in `package.json` never gets updated. Relay [reads the Git ref](https://github.com/mozilla/fx-private-relay/blob/8e6e6a4eb3c6f7bf6868ee5eb0ee2082b9ace021/privaterelay/views.py#L107-L112), maybe we could do that too?

Reply:
Yes, circle does that for us automatically